### PR TITLE
Add page for actions and issues

### PR DESCRIPTION
**Pull request by U8NWXD**

### CODE REVIEW

File: Actions-and-issues.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+These actions are recorded using the functions provided by the [stats reporting service](https://github.com/oppia/oppia/tree/develop/core/templates/pages/exploration-player-page/services/stats-reporting.service.ts). For example, when a learner transitions into a terminal state (a card that ends the exploration), the [conversation skin directive](https://github.com/oppia/oppia/tree/develop/core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts) calls `StatsReportingService.recordExplorationCompleted()`.
+
+When a learner's journey through an exploration goes smoothly, we don't store the actions anywhere; however, when we detect a problem in a learner's journey, then we create an issue that stores the associated actions.
+
+## Issues
+
+We define three different issues at [`extensions/issues/`](https://github.com/oppia/oppia/tree/develop/extensions/issues):
+
+* `MultipleIncorrectSubmissions`: This issue is recorded when a user submits 3 or more incorrect answers to an interaction. It stores the name of the state where the user provided incorrect answers and the number of incorrect answers that were submitted.
```
Comment (U8NWXD) [inline Actions-and-issues.md:32]:
```suggestion
* `MultipleIncorrectSubmissions`: This issue is recorded when a user submits 3 or more incorrect answers to an interaction. It stores the name of the state where the user provided incorrect answers and the number of incorrect answers that they submitted.
```

Comment (U8NWXD) [reply Actions-and-issues.md]: done

File: Actions-and-issues.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* [Introduction](#introduction)
+* [Actions](#actions)
+* [Issues](#issues)
+* [Using actions and issues](#using-actions-and-issues)
+
+## Introduction
+
+As a user plays through an exploration, we want to track their progress to identify any problems. If we do find problems with the exploration, we can raise those to the exploration creator. For example, if a user gets stuck in a cycle of cards or submits many incorrect answers to an interaction, the exploration could probably be improved.
+
+We use store the steps a user takes through an exploration as _actions_, and we create _issues_ to keep track of problems like many incorrect answers. Both actions and issues are defined as [[schemas|Schemas]].
```
[the remark this answers is not in the document]
Comment (U8NWXD) [reply Actions-and-issues.md]: done

File: _Sidebar.md [hunk 3]
```diff
@@ -99,6 +99,7 @@
     * [[Objects|Creating-Objects]]
     * [[Dependencies|Creating-Dependencies]]
     * [[Value generators|Creating-Value-Generators]]
+    * [[Actions and isues|Actions-and-issues]]
```
[the remark this answers is not in the document]
Comment (U8NWXD) [reply _Sidebar.md]: done

**Comment by U8NWXD** [comment]
@DubeySandeep @brianrodri PTAL

**Review by brianrodri — APPROVED** [review]
It's perfect!!

**Review by DubeySandeep — APPROVED** [review]

**Comment by U8NWXD** [comment]
@seanlip PTAL

File: Actions-and-issues.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+* `ExplorationQuit`: This action is recorded when a learner leaves an exploration. It stores the name of the state at which the learner quit and how long the user spent in the last state they visited.
+
+These actions are recorded using the functions provided by the [stats reporting service](https://github.com/oppia/oppia/tree/develop/core/templates/pages/exploration-player-page/services/stats-reporting.service.ts). For example, when a learner transitions into a terminal state (a card that ends the exploration), the [conversation skin directive](https://github.com/oppia/oppia/tree/develop/core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts) calls `StatsReportingService.recordExplorationCompleted()`.
+
+When a learner's journey through an exploration goes smoothly, we don't store the actions anywhere; however, when we detect a problem in a learner's journey, then we create an issue that stores the associated actions.
```
Comment (seanlip) [inline Actions-and-issues.md:26]: Maybe add, somewhere: "Note that we do not store any details about the learner's identity."

File: Actions-and-issues.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+  This issue stores a list of the state names that make up the cycle.
+
+* `EarlyQuit`: This issue is recorded whenever a user quits an exploration after fewer than 300 seconds. It records the name of the state where the user quit and the time they spent on the exploration.
+
+  The logic behind this cycle detection lives in the `EarlyQuitTracker` class in the [playthrough service](https://github.com/oppia/oppia/tree/develop/core/templates/services/playthrough.service.ts).
+
+## Using actions and issues
+
+Right now, neither issues nor their associated actions are surfraced to exploration creators. They used to be available through the improvements tab of the exploration editor, but that tab has been removed. We are currently working on building a new improvements tab.
```
Comment (seanlip) [inline Actions-and-issues.md:51]: "surfaced" is misspelled

**Review by seanlip — CHANGES_REQUESTED** [review]
Looks great! Just a couple of notes.

File: Actions-and-issues.md (hunk 4 above)
Comment (U8NWXD) [reply Actions-and-issues.md]: done

**Comment by U8NWXD** [comment]
@seanlip PTAL

**Review by seanlip — CHANGES_REQUESTED** [review]

File: Actions-and-issues.md [hunk 6]
```diff
@@ -23,7 +23,7 @@ There are three actions defined at [`extensions/actions/`](https://github.com/op
 
 These actions are recorded using the functions provided by the [stats reporting service](https://github.com/oppia/oppia/tree/develop/core/templates/pages/exploration-player-page/services/stats-reporting.service.ts). For example, when a learner transitions into a terminal state (a card that ends the exploration), the [conversation skin directive](https://github.com/oppia/oppia/tree/develop/core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts) calls `StatsReportingService.recordExplorationCompleted()`.
 
-When a learner's journey through an exploration goes smoothly, we don't store the actions anywhere; however, when we detect a problem in a learner's journey, then we create an issue that stores the associated actions.
+When a learner's journey through an exploration goes smoothly, we don't store the actions anywhere; however, when we detect a problem in a learner's journey, then we create an issue that stores the associated actions. Note that the issues does not record the user's identity.
```
[the remark this answers is not in the document]
Comment (U8NWXD) [reply Actions-and-issues.md]: done

**Comment by U8NWXD** [comment]
@seanlip PTAL

**Review by seanlip — APPROVED** [review]
Thanks, LGTM

**Pull request merged**